nautobot/extras/tests/test_jobs.py

```diff
@@ -1019,6 +1020,11 @@ class ScheduledJobIntervalTestCase(TestCase):
     # datetime weekday starts on Monday (Sunday = 6)
     datetime_days = ["Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday", "Sunday"]
 
+    def _strptime(self, time_string):
+        """Return a utc datetime object from a string with the format: 01 Jan 2024 09:00AM"""
+        time_format = r"%d %b %Y %I:%M%p"
+        return datetime.datetime.strptime(time_string, time_format).replace(tzinfo=timezone.utc)
```

Comment:
Is the `.replace` converting the datetime *from* an initial `America/New_York` timezone to UTC? Trying to understand why we're using UTC here instead of `America/New_York`.

Reply:
We're using utc because that's what `django.utils.timezone.now` returns, which is required to fix this issue. The default is to use celery's `app.now` which returns a datetime in the local timezone. The `.replace` is adding the timezone without datetime trying to convert it from the system's local time to utc.